### PR TITLE
contentful basic api set up

### PR DESCRIPTION
**Pull request by thinklikeadesigner**
bare bones api laid out

### CODE REVIEW

File: src/components/App.js
Comment (McCambley) [inline src/components/App.js:23]: should this just run once on initial App render with an empty dependency array? Does that matter?

File: src/components/App.js
Comment (McCambley) [inline src/components/App.js:5]: Could we slot this file in a /utils folder instead of the components folder?

File: src/components/App.js
Comment (McCambley) [inline src/components/App.js:16]: YOU MAKE IT LOOK SO EASY
Wow.. so cool. We're connected now.

File: src/components/client.js
Comment (McCambley) [inline src/components/client.js:3]: There it is, amazing. I'll watch that video you linked about setting up a blog before filling this 9 lines of code with a billion questions, but let it be known that I understand very little of what's going on... some... but very little. That being said, what I do understand is that this is great. Lot's going on behind the scenes, and we're connected! Awesome work Rebecca!

**Review by McCambley — CHANGES_REQUESTED** [review]
Hey Rebecca, this is incredible! Just two comments on file structure and the useEffect call. I'll defer to your judgement as to whether or not changes need making. Very very cool.

File: src/components/client.js
Comment (McCambley) [inline src/components/client.js:9]: I may have spoke too soon, when I try to compile the code with `npm run start` I recieve an error `TypeError: Expected parameter accessToken`. I'll attach a screenshot below.

**Comment by cjmaret** [comment]
@thinklikeadesigner I wish I could review it, but I know probably 10% of what Jake knows about this... so I will not be of help 😞  I literally put off the Practicum lessons on setting up a server to work on the Profession Hero today 😅

**Comment by McCambley** [comment]
We're holding off on this while @thinklikeadesigner sets up a Graphql version of our query process

**Pull request closed without merging**
